stablehlo/tests/ops_stablehlo_roundtrip.mlir

```diff
@@ -1,7 +1,4 @@
 // RUN: stablehlo-opt %s | stablehlo-opt
-// RUN: diff <(stablehlo-opt %s) <(stablehlo-opt -emit-bytecode %s | stablehlo-opt)
-// RUN: stablehlo-opt -emit-bytecode -debug-only=stablehlo-bytecode %s 2>&1 | (! grep 'Not Implemented')
-// RUN: stablehlo-opt -emit-bytecode %s | stablehlo-opt -debug-only=stablehlo-bytecode 2>&1 | (! grep 'Not Implemented')
```

Comment:
Can you tell more about the motivating for disabling these RUN directives? cc @GleasonK for context since he added them here.

Reply:
My mistake: I was debugging something, restored now!